scripts/generator/graphql_generator/csharp/SDK/iOS/iOSWebCheckout.cs.erb

```diff
@@ -0,0 +1,43 @@
+namespace <%= namespace %>.SDK.iOS {
+#if !SHOPIFY_MONO_UNIT_TEST
```

Comment:
If I put it outside the project won't compile because there is no other place in the project where we declare the SDK.iOS namespace. Should I define an empty namespace somewhere else or leave the check inside?